netci.groovy

```diff
@@ -1987,22 +1992,65 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                     def buildArchitecture = 'arm'
 
+                    def buildOpts = ''
+
                     // For 'armlb' (the JIT LEGACY_BACKEND architecture for arm), tell build.cmd to use legacy backend for crossgen compilation.
                     // Legacy backend is not the default JIT; it is an aljit. So, this is a special case.
-                    def armCrossgenOpt = ''
                     if (architecture == 'armlb') {
-                        armCrossgenOpt = '-crossgenaltjit legacyjit.dll'
+                        buildOpts += ' -crossgenaltjit legacyjit.dll'
+                    }
+
+                    if (enableCorefxTesting) {
+                        // We shouldn't need to build the tests. However, run-corefx-tests.py currently depends on having the restored corefx
+                        // package available, to determine the correct corefx version git commit hash, and we need to build the tests before
+                        // running "tests\\runtest.cmd GenerateLayoutOnly". So build the pri-0 tests to make this happen.
```

Comment:
Would #15673 make this better?  If so, should we merge that and make this depend on it?

Reply:
yes. Note that we already have a dependency on this for corefx testing on all platforms (it's not new with this change).

Reply:
It sounds like the consensus is that #15673 should be merged?